microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -534,6 +534,36 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image<!-- todo: is the menuitem capital I or lowercase?  global in section -->
+
+To use the Magnify Image feature, right-click an image and then select **Magnify Image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
+
+
+<!-- ------------------------------ -->
+#### AI-based image enhancement
+
+If your browser has WebNN enabled, there's also a button<!-- todo: what's the button label?  can we commit to specifying the button label at this time?  [click] the **Foo** button --> to use AI to enhance the image.  To check whether WebNN is enabled, go to the `edge://gpu` page, and then in the **Graphics Feature Status** section, see whether **WebNN** says **Enabled**:<!-- todo: how to control whether WebNN is enabled? -->
+
+![The status of the WebNN setting, shown in the edge://gpu page](./index-images/webnn-disabled-gpu.png)
+
+AI-based image enhancement is done on your device by using a model that's downloaded from Microsoft servers when you click the AI enhancement<!-- todo: what's the button label?  can we commit to specifying the button label at this time?  [click] the **Foo** button --> button.  Neither the image nor any personally identifiable information is sent to Microsoft.
+
+
+<!-- ------------------------------ -->
+#### Turning off the feature
+
+To turn off the Magnify Image feature, use the command-line flag `msEdgeImageMagnifyUI`.
+<!-- todo: state the gui nav path, if any: eg: 
+; select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Sidebar** > **App and notification settings** > **Foo**, and then use the **Foo** toggle.
+-->
+
+To turn off the **Ctrl-Ctrl** keyboard shortcut, use `edge://settings`.
```

Comment:
where in edge Settings can you turn off Ctrl-Ctrl shortcut?
@sjdallst2 

rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

Reply:
edge://settings/content/MagnifyImage will take you directly to the setting.

Reply:
result:
```
To turn off the **Ctrl-Ctrl** keyboard 
shortcut, go to `edge://settings/content/MagnifyImage`.  
Or, select **Settings and more** (![Settings and more 
icon](./index-images/settings-and-more-icon.png)) 
> **Cookies and site permissions** > **Magnify image**, 
and then use the **Enable Magnify keyboard shortcut** 
toggle.
```

Reply:
it seems practical to give both the shortcut url and the gui nav path - seems robust, lots of clues for future maintainers & users, painting an adequate picture, w/o having to show screenshot
![image](https://github.com/user-attachments/assets/40db60d5-88c8-4590-b379-732c125b7cce)
I'd want a screenshot before that anyway, if we do screenshots of Settings for nav:
![image](https://github.com/user-attachments/assets/4928b39b-9e8c-48af-bbb8-8aaa25758e02)

yet another approach is to use Search box in upper left - that approach is probably not used (described) in this Privacy article, but I'll consider, in a later global update.